.github/workflows/publish_coverage.yml

```diff
@@ -36,11 +48,13 @@ jobs:
           S3_KEY: "coverage_badge_total.svg"
           AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID_STAGING }}
           AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY_STAGING }}
+          AWS_AWS_ACCOUNT: ${{ secrets.AWS_ACCOUNT_STAGING }}
           AWS_REGION: "us-east-1"
```

Comment:
This Deploy coverage to S3 won't work anymore since it requires access and secret keys, one thing that we don't use anymore